src/score/chordnamecontext.cpp

```diff
@@ -0,0 +1,154 @@
+/*!
```

Comment:
Um... why is this file shown in your PR? Shouldn't your branch be rebased to the latest master?